src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -25,6 +25,23 @@ public Tag(String tagName) {
         this.tagName = tagName;
     }
 
+    public String getTagName() {
+        return this.tagName;
+    }
+
+    /**
+     * Returns true if both tags have the same tag name.
+     * This defines a weaker notion of equality between two tags.
+     */
+    public boolean isSameTag(Tag otherTag) {
```

Comment:
not sure if this method is redundant since we already have the `equals` method? since different tag object but same name attribute is still considered equal?

Reply:
Yup, I agree with the latter, but the method was included since it's faster and easier to just check if the tag names are the same rather than invoking the equals method to check if the object is the same